main/src/cgeo/geocaching/models/Geocache.java

```diff
@@ -398,7 +400,7 @@ public boolean gatherMissingFrom(final Geocache other) {
             searchContext = other.searchContext;
         }
 
-        this.eventTimeMinutes = null; // will be recalculated if/when necessary
+        this.eventStartTimeInMinutes = null; // will be recalculated if/when necessary
```

Comment:
Should we reset the end time as well (here and at all other places where eventStartTimeInMinutes gets cleared)?

Reply:
end time is only used together with start time. So while it might happen that only start time is reset before accessing end time both start and end time will be recalculated. So not necessary to clean it up.

Reply:
This is totally true as of now, however keep in mind that the code base might change over time. Parameters IMHO shouldn't magically depend on each other, either reset both or create a very small data class holding both values.